modules/custom/az_paragraphs/src/Plugin/paragraphs/Behavior/AZTextWithMediaParagraphBehavior.php

```diff
@@ -237,17 +264,40 @@ public function preprocess(&$variables) {
           $content_classes[] = HTML::getClass($spacing_prefix . $config['text_media_spacing']);
       }
     }
-
+    if ($config['style'] === 'bottom') {
+      if ($config['bg_color'] === 'light') {
+        $key = array_search('light', $content_classes);
+        unset($content_classes[$key]);
+        $content_classes[] = 'bg-white';
+        $content_classes[] = 'shadow';
+        $content_classes[] = 'mb-4';
+      }
+      elseif ($config['bg_color'] === 'dark') {
+        $key = array_search('dark', $content_classes);
+        unset($content_classes[$key]);
+        $content_classes[] = 'bg-black';
+        $content_classes[] = 'shadow';
+        $content_classes[] = 'mb-4';
+      }
+    }
     // Set content classes.
     $variables['elements']['#fieldgroups']['group_az_content']->format_settings['classes'] = implode(' ', $content_classes);
+    // Set title element if a heading level other than h2 (the default) was selected.
+    if ($config['title_level'] !== 'h2') {
+      $variables['elements']['#fieldgroups']['group_az_title']->format_settings['element'] = $config['title_level'];
+    }
     // Get title classes.
     $title_classes = [
       'mt-0',
       'bold',
+      HTML::getClass($config['title_alignment']),
     ];
     if (!empty($config['bg_color']) && $config['bg_color'] !== 'dark') {
       $title_classes[] = 'text-blue';
     }
+    if ($config['title_level'] !== 'h1' && $config['title_level'] !== 'h2') {
```

Comment:
This is not intuitive for a user using the Title heading level options. Since it presents h1-h6, if I want it to be h3, I expect the styling to show h3, not h2.

Reply:
@joshuasosa The intent of this change is to allow different heading levels for maximum accessibility compliance while reducing disruption to the existing visual style. If other visual styles outside the brand-approved styles on the title field are desired, the body field should be used to get the desired look.

Reply:
I disagree. The current wording does not make that clear. It looks like if I can set it to h6, I expect it to be h6.

> If other visual styles outside the brand-approved styles on the title field are desired, the body field should be used to get the desired look.

If that's the case, then this heading styling is an unnecessary change. You can use the body field to achieve your goal.

Reply:
I agree, we shouldn't override styling too much for consistency across the entire page.

Reply:
@danahertzberg From our discussion in today's AZ Digital PR review, we thought it may be best to hold off on removing the `text-blue` class. Since that could be considered a significant visual change, we might want to create a separate PR to set that title color as the default and allow users to choose the title color. We can discuss further when you are ready: for right now, I will go ahead and remove the h2 class.

Reply:
That sounds good! Yeah I think to remove text-blue, a dropdown selection should be provided to apply a color, including setting blue as the default in the dropdown. Removing h2 class and optionally setting h2 as the default in the new dropdown is good for now. Thanks!